src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -23,6 +23,16 @@ internal sealed class GitHubAccessor
     {
         private static readonly Uri s_url = new("https://api.github.com/graphql");
 
+        private static readonly HttpClient s_sharedHttpClient =
```

Comment:
do we need to set this for all `HttpClient`?
For instance, https://github.com/dotnet/docfx/blob/be1e9e78e739e9a9d3f71eee1b96346e6178091b/src/docfx/config/ops/OpsAccessor.cs#L46

Reply:
Thanks for reminder this, find all to fix.